project/osprojects/tests.py

```diff
@@ -1,3 +1,11 @@
-from django.test import TestCase
+import pytest
+from .factories import OSProjectsFactory
 
-# Create your tests here.
+
+@pytest.mark.django_db
+def test_osprojects_model():
+    """ Test OSProjects model """
+    # create OSProjects model instance
+    osproject = OSProjectsFactory(title="Buddybot", guid="cc63f918-d515-11ea-956f-0242ac130002")
+    assert osproject.title == "Buddybot"
```

Comment:
Note: was thinking I could add more asserts here, but was wondering if this might also be fine

Reply:
Up to you.  I usually go for an entire record -- or even two, then pick up to four things to assert.  But even one proves it was inserted into the DB, which is what we are going for here....so.....